### DISCUSSION TITLE
Need Sort by option in bookmarked list

### DISCUSSION THREAD

**Discussion by gvgvgvijayan** [dailydotdev/daily#742] [Feature Requests]
Shall we have sorting option in bookmarked page because it would be helpful when we want read an article based on highest upvote to the lowest one instead of bookmarked sequence/date order.

- Sort by Upvote count
- Sort by Comment count
- Sort by Bookmarked date
![Screenshot from 2022-10-14 16-30-11](https://user-images.githubusercontent.com/4319097/195832248-d095d1f5-7624-447d-a59c-d71ed9737b18.png)

Thanks for the nice product. Learned many things from daily.dev :)

**Comment by idoshamun** [comment]
Hi @gvgvgvijayan,

Thanks for suggesting it. Let me log that down :)

**Reply by gvgvgvijayan** [reply]
Cool! Thanks :)
